docs/docker.md

```diff
@@ -0,0 +1,42 @@
+# Running in docker  
+  
+Run docker image from gchr.io/patrickdappollonio/kubectl-slice
+
+## Usage
+The container is build to execute the the kubectl-slice tool inline. 
+   
+Example:  
+`docker run --rm -v "${PWD}/slice/testdata":workdir kubectl-slice -f ingress-namespace.yaml -o ./`  
```

Comment:
This could potentially be better in a multi-line where people can hit "copy" on Github's UI, with triple backticks before and after.